bazel-scf-v2

```diff
@@ -0,0 +1 @@
+/home/toaster/.cache/bazel/_bazel_toaster/0291e00957d43be0bd77a17ccdc76a11/execroot/scf
```

Comment:
This symlink slipped out the generated Bazel directories. We need to update the `.gitignore` to include `/bazel-*`.